test/is-promise.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+/* global describe, it */
+
+const { expect } = require('chai')
+const isPromise = require('../lib/is-promise')
+
+describe('isPromise', () => {
```

Comment:
Could you please add a test to check that for a non-falsy scalar value isPromise returns false?

This is IMHO one of the main use cases.

Reply:
Sure. I added a test on a string value